### PR TITLE
IGAPP-1195: Use poi metadescription from cms

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Nice, tested in Firefox.

Does this mean we can remove the excerpt or do we need to keep that for backward compatibility?

File: api-client/src/endpoints/createPOIsEndpoint.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ export default (baseUrl: string): Endpoint<ParamsType, Array<PoiModel>> =>
               thumbnail: poi.thumbnail,
               availableLanguages: mapAvailableLanguages(poi.available_languages),
               excerpt: poi.excerpt,
+              metaDescription: poi.meta_description ? poi.meta_description : null,
```
Comment (SteffiStuffel) [inline api-client/src/endpoints/createPOIsEndpoint.ts:34]: You can just write `metaDescription: poi.meta_description ?? null`

**Review by SteffiStuffel — APPROVED** [review]
Tested in Chrome.

File: api-client/src/endpoints/createPOIsEndpoint.ts (hunk 1 above)
Comment (sarahsporck) [reply api-client/src/endpoints/createPOIsEndpoint.ts:34]: Does not work as the meta description is an empty string if it is not set and `"" ?? null` resolves to ""

**Comment by sarahsporck** [comment]
> Nice, tested in Firefox.
> 
> Does this mean we can remove the excerpt or do we need to keep that for backward compatibility?

excerpt != meta_description.

> Nice, tested in Firefox.
> 
> Does this mean we can remove the excerpt or do we need to keep that for backward compatibility?

Excerpt seems to be different from the meta description.

**Pull request merged**